src/test/resources/pages/InstructorCourseDetailsEmptyCourse.html

```diff
@@ -335,7 +332,7 @@
                 </div>
                 
                 <p class="form-group text-muted">
-                    The default visibility for your comment is private. You may change it using the ‘show visibility options’ button above.
+                    The default visibility for your comment is private. You may change it using the 'show visibility options' button above.
```

Comment:
They're typographically correct.
Quotation marks are supposed to be curved; limitations in the past created the straight ones.
There were then attempts to "correct" this by having word processors autocorrect them to the curved version.

That said, in this case, I'm only mentioning it since it was used previously rather than out of any personal opinion which looks better.